extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -65,13 +70,16 @@ export class RemoteQueriesManager {
 
       const queryResult = this.mapQueryResult(executionEndTime, resultIndex);
 
+      // Kick off auto-download of results.
+      void commands.executeCommand('codeQL.autoDownloadRemoteQueryResults', queryResult);
```

Comment:
Nice. This will run in the background with all the error handling.

Reply:
Yep! I do wonder if I'm abusing the extension API a bit, but it seems to work out well for us so far, and I haven't thought of a good reason not to do that.

Reply:
I think this is _exactly_ the right use for commands. Otherwise, we'd have to do a bunch of fancy error handling.